gdal/GDALmake.opt.in

```diff
@@ -459,6 +459,16 @@ RASDAMAN_ENABLED  =       @RASDAMAN_ENABLED@
 RASDAMAN_INC  =   @RASDAMAN_INC@
 RASDAMAN_LIB  =   @RASDAMAN_LIB@
 
+
+#
+# BRUNSLI (JPEG XL)
```

Comment:
should be set by configure.ac

Reply:
That's the part I'm not familiar with, I'll see if I can figure it out.

Reply:
For JXL, I could point at https://github.com/rouault/gdal/blob/jxl/gdal/configure.ac#L5970 and https://github.com/rouault/gdal/blob/jxl/gdal/GDALmake.opt.in#L561 :-)